templates/lightning/channel_management.html

```diff
@@ -58,17 +61,20 @@ <h2>Channels</h2>
         <li><strong>Age:</strong> n/a</li>
         <li><strong>Peer:</strong> <small class="break-all">{{ channel.peer.id }}</small></li>
         <li>
+        {% if channel.peer.connected %}
```

Comment:
we might still want to close the channel even if the peer is not connected.

Reply:
using channel.state instead of channel.peer.connected.